dtls.c

```diff
@@ -4267,6 +4267,7 @@ dtls_handle_message(dtls_context_t *ctx,
   }
 
   while ((rlen = is_record(msg,msglen))) {
+
```

Comment:
Why an empty line? I think this should be removed before merge.

Reply:
OK.